### ISSUE TITLE
[distributed] BenchmarkListObjects fails to run 

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#3133]
```
BenchmarkGetObjectParallel25MbXL-12                 3000            540933 ns/op          102033 B/op        630 allocs/op
BenchmarkGetObjectParallel50MbFS-12                 2000           1024124 ns/op          101974 B/op        630 allocs/op
BenchmarkGetObjectParallel50MbXL-12                 2000           1042291 ns/op          101968 B/op        630 allocs/op
--- FAIL: BenchmarkListObjects
        object-api-listobjects_test.go:579: Unexpected err:  Unable to recognize backend format, unformatted disk found
BenchmarkPutObjectPart5MbFS-12                         2         753799389 ns/op        522589848 B/op    117298 allocs/op
BenchmarkPutObjectPart5MbXL-12                         2         778404900 ns/op        523245472 B/op    117304 allocs/op
BenchmarkPutObjectPart10MbFS-12                        2         843387361 ns/op        687712760 B/op     57648 allocs/op
BenchmarkPutObjectPart10MbXL-12                        2         860397126 ns/op        685615456 B/op     57642 allocs/op
BenchmarkPutObjectPart25MbFS-12                        2         957281432 ns/op        534818672 B/op     29430 allocs/op
BenchmarkPutObjectPart25MbXL-12                        2        1001232957 ns/op        534425360 B/op     29428 allocs/op
BenchmarkPutObjectPart50MbFS-12                        2         870251882 ns/op        568877816 B/op     16161 allocs/op
```
## Expected Behavior
Benchmarks should run fine.

## Current Behavior
Benchmarks do not run or skip few of them.

## Possible Solution
Any backend format needed should be initialized properly before the tests are run.

## Steps to Reproduce (for bugs)

Just run 
```
go test -run=NONE -bench. 
```

## Context
The problem is a blocker for our ongoing effort in getting benchmarking automation in place.

**Comment by hackintoshrao** [comment]
Fixing this.

**Comment by hackintoshrao** [comment]
This issue is fixed.

**Comment by harshavardhana** [comment]
Fixed by #3142
